galaxy_mobile/lib/utils/switch_page_helper.dart

```diff
@@ -26,27 +25,26 @@ class SwitchPageHelper {
       int pageSize,
       bool muteOtherCams,
       OnSwitchVideosCallback onSwitchVideosCallback) {
-    this.widget = widget;
-    pageSize = pageSize;
-    muteOtherCams = muteOtherCams;
-    makeSubscription = makeSubscription;
```

Comment:
why this field was removed? and also the one below?

Reply:
I modified to use "this." prefix and changed the names of some of them.
I removed widget field and added onSwitchVideosCallback instead.
The 6 fields are still there